backend/core/rule/tree.py

```diff
@@ -0,0 +1,72 @@
+import yaml
+from core.rule.functions import *
+
+
+class TreeLoader(object):
+    def __init__(self, rule_name, sugang_list):
+        f = open(rule_name, 'r', encoding='utf-8')
```

Comment:
I will change this entire file loading code later using function (because rule yaml file will be increase, so we have to keep these in another directory, not current directory)